man/tpm2_policycommandcode.1.md

```diff
@@ -0,0 +1,76 @@
+% tpm2_policycommandcode(1) tpm2-tools | General Commands Manual
+%
+% JANUARY 2018
+
+# NAME
+
+**tpm2_policycommandcode**(1) - Restrict TPM object authorization to specific
+TPM commands or operations.
+
+# SYNOPSIS
+
+**tpm2_policycommandcode** [*OPTIONS*]
+
+# DESCRIPTION
+
+**tpm2_policycommandcode**(1) Restricts TPM object authorization to specific
+TPM commands or operations. Useful when you want to allow only specific commands
+with the TPM object. Requires support for extended sessions with resource manager.
+
+# OPTIONS
+
+  * **-S**, **--session**=_SESSION\_FILE_:
+
+    A session file from **tpm2_startauthsession**(1)'s **-S** option.
+
+   * **-o**, **--policy-file**=_POLICY\_FILE_:
+
+    File to save the policy digest.
+[common options](common/options.md)
+
+[common tcti options](common/tcti.md)
+
+# EXAMPLES
+
+Start a *policy* session and extend it with a specific command like unseal.
+Attempts to perform other operations would fail.
+
+## Create an unseal-only policy
+```
+TPM_CC_UNSEAL=0x15E
+tpm2_startauthsession -S session.dat
+tpm2_policycommandcode -S session.dat -c $TPM_CC_USEAL -o policy.dat
```

Comment:
It's an argument now, example is broken. Move argument to last item in command line vector.